src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="image">
+        <a href="index.html">
+          <img
+            src="images/logo.png"
+            alt="logo_moyo"
+          />
```

Comment:
The `alt` attribute should describe the image content, not just be the name of the company. It's better to describe what the logo looks like or the company's full name. For example, 'Moyo Company Logo'.